editor/post-format/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { find, flowRight } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withInstanceId } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEditedPostAttribute, getSuggestedPostFormat } from '../selectors';
+import { editPost } from '../actions';
+
+const POST_FORMATS = [
+	{ id: 'aside', caption: __( 'Aside' ) },
+	{ id: 'gallery', caption: __( 'Gallery' ) },
+	{ id: 'link', caption: __( 'Link' ) },
+	{ id: 'image', caption: __( 'Image' ) },
+	{ id: 'quote', caption: __( 'Quote' ) },
+	{ id: 'standard', caption: __( 'Standard' ) },
+	{ id: 'status', caption: __( 'Status' ) },
+	{ id: 'video', caption: __( 'Video' ) },
+	{ id: 'audio', caption: __( 'Audio' ) },
+	{ id: 'chat', caption: __( 'Chat' ) },
+];
+
+function PostFormat( { onUpdatePostFormat, postFormat = 'standard', suggestedFormat, instanceId } ) {
+	const postFormatSelectorId = 'post-format-selector-' + instanceId;
+	const suggestion = find( POST_FORMATS, ( format ) => format.id === suggestedFormat );
+
+	// Disable reason: A select with an onchange throws a warning
```

Comment:
Existed previously but... "it's a warning" is not a reason to disable a warning.